ts/Series/Wordcloud/WordcloudSeries.ts

```diff
@@ -0,0 +1,691 @@
+/* *
+ *
+ *  Experimental Highcharts module which enables visualization of a word cloud.
+ *
+ *  (c) 2016-2020 Highsoft AS
+ *  Authors: Jon Arild Nygard
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ * */
+
+'use strict';
+
+import type BBoxObject from '../../Core/Renderer/BBoxObject';
+import type PositionObject from '../../Core/Renderer/PositionObject';
+import type SizeObject from '../../Core/Renderer/SizeObject';
+import type { StatesOptionsKey } from '../../Core/Series/StatesOptions';
+import type SVGAttributes from '../../Core/Renderer/SVG/SVGAttributes';
+import type SVGElement from '../../Core/Renderer/SVG/SVGElement';
+import type {
+    WordcloudSeriesOptions,
+    WordcloudSeriesRotationOptions
+} from './WordcloudSeriesOptions';
+import BaseSeries from '../../Core/Series/Series.js';
+import ColumnSeries from '../Column/ColumnSeries.js';
```

Comment:
Here the same. Because world cloud is its own module, you have to import series dependencies over `BaseSeries`, to not increase the product JS file size.